api-reference/50 Common/utils/pdfExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,226 @@ module: pdf_exporter
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports grid data to a PDF file.
 
 ##### return: Promise<void>
-<!-- Description goes here -->
+A <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">Promise</a> that is resolved with an object that contains the coordinates of the first and last cells.
 
 ##### param(options): PdfExportDataGridProps
-<!-- Description goes here -->
+Export settings.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This method requires the <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library to export data and the <a href="https://github.com/simonbengtsson/jsPDF-AutoTable" target="_blank">jsPDF-AutoTable</a> plugin to create tables in exported files.
+
+You can call this method at any point in your application.
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        $('#exportButton').dxButton({
+            // ...
+            onClick: function() {
+                const doc = new jsPDF();
+                DevExpress.pdfExporter.exportDataGrid({
+                    jsPDFDocument: doc,
+                    component: dataGrid
+                }).then(function() {
+                    doc.save('Customers.pdf');
+                });
+            }
+        });
+
+        let dataGrid = $('#gridContainer').dxDataGrid({
+            // Specify grid options here
```

Comment:
Please don't abbreviate widget names. Anyway, readers already configured the widget, so we shouldn't ask them to do it again.

```suggestion
            // ...
```